### ISSUE TITLE
Add unit tests for yeast models

### ISSUE THREAD

**Issue by MichaelWashburnJr** [BrewCenter/BrewCenterAPI#55]

**Comment by pietscham** [comment]
Hi, mind if I take a crack at this one? Would this just entail BrewCenterAPI/brewcenter_api/brew_data/models/yeast.py?

**Comment by MichaelWashburnJr** [comment]
Yes, just the yeast-related models. Let me know if you're still interested - I've been out of town for a while and haven't had a chance to monitor this github.

**Comment by pietscham** [comment]
Yes, definitely still interested!

**Comment by pietscham** [comment]
@MichaelWashburnJr Sorry for not getting this done, went on vacation for a couple of weeks. I'll have a PR ready within the next day, if still possible!

**Comment by MichaelWashburnJr** [comment]
Yep go right ahead! Thanks!
